k8s/demo/common/ccd/ccd-demo-latest-chart.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: ccd-demo-latest-chart
+  namespace: ccd
+  annotations:
+    flux.weave.works/automated: 'true'
+    flux.weave.works/tag.java: 'glob:prod-*'
+spec:
+  releaseName: ccd-demo-latest-chart
+  timeout: 900
+  forceUpgrade: true
+  rollback:
+    enable: true
+    force: true
+  chart:
+    git: 'https://github.com/hmcts/chart-ccd'
+    ref: master
+    path: ccd
+  values:
+    ccd-api-gateway-web:
+      enabled: true
+      nodejs:
+        ingressHost: gateway-ccd-aks.service.core-compute-demo.internal
+    idam-pr:
+      enabled: true
```

Comment:
double check but I think the way you enable idam-pr is with tags

Reply:
we have to try this on demo and see